### DISCUSSION TITLE
Installed SeisSol but not running

### DISCUSSION THREAD

**Discussion by Jing-Lee-tj** [SeisSol/SeisSol#974] [Q&A]
Hello Guys.
I have followed the instruction of Compiling and running SeisSol and skip things related to GPUs.
After running "make -j8 install" I get SeisSol_Release_dskx_4_elastic in my build directory.
![4ae5336fdd65765657695f2ec1a9ebe](https://github.com/SeisSol/SeisSol/assets/35103101/459f26ac-0e16-434d-8da8-b9afd7bf38ee)
When I try to run the first example, I copy the /Examples/tpv33 and copy the SeisSol_Release_dskx_4_elastic to this directory, and running
mpiexec -np 8 ./SeisSol_Release_dskx_4_elastic parameters.par, it shows errors:
![d2b02589a629b855f81e9a067d61cad](https://github.com/SeisSol/SeisSol/assets/35103101/966a87a5-16d9-4aa3-b40a-4ee4a562d7e5)
I have no ideas about what happend? Did I failed to install the SeisSol or the the way I running it have mistakes?

**Comment by krenzland** [comment]
Without any data about your software stack and machine, we cannot help you. As a guess, try to compile with -DHOST_ARCH=snb, as signal 4 indicates an illegal instruction.

**Comment by Jing-Lee-tj** [comment]
Sorry, I just forget to run the "make" command.
It seems I have installed SeisSol correctly.
But I have been confronted with anothoer problem: 
![1697211437946](https://github.com/SeisSol/SeisSol/assets/35103101/abad5166-7812-4444-b279-384d022816c6)
My computer is windows with only one cpu and I am currently trying to run SeisSol on the Windows Subsystem for Linux (WSL). Does this error information mean that I need to install a linux system to use SeisSol?
![1697211244792](https://github.com/SeisSol/SeisSol/assets/35103101/9c5a3ad3-b115-41eb-99d8-463843733fc9)

**Reply by davschneller** [reply]
If you do not use MPI (or single-node), then compiling without communication thread should solve it. I.e. `-DCOMMTHREAD=OFF`.
I'm sorry about the "no free CPUs left" issue, it's a bug right now—single-rank computations will not use a communication thread anyways.

Also, you may set `-DHOST_ARCH=hsw` for a slightly more optimized performance, based on the CPU model you have. Though `-DHOST_ARCH=snb` may also already give a decent performance.
